timemachine/fe/utils.py

```diff
@@ -65,7 +67,7 @@ def convert_uM_to_kJ_per_mole(amount_in_uM):
         Binding potency in kJ/mol.
 
     """
-    return 0.593 * np.log(amount_in_uM * 1e-6) * 4.18
+    return 0.593 * np.log(amount_in_uM * 1e-6) * constants.KCAL_TO_KJ
```

Comment:
Based on prior notes, I think 0.593 is intended to be `molar gas constant * temperature`, in units of kcal/mol, but
(1) the temperature may be inconsistent with `DEFAULT_TEMP = 300` kelvin
(2) there may be precision loss.

https://www.wolframalpha.com/input?i=molar+gas+constant+*+300+kelvin = 0.5962 kcal/mol
https://www.wolframalpha.com/input?i=molar+gas+constant+*+298+kelvin = 0.5922 kcal/mol

Reply:
`RT` is from the table

https://en.wikipedia.org/wiki/KT_(energy) (measured in kcal/mol)

Note that the `T` here should probably be consistent with experimental temperatures, not simulation temperatures, if converting labelled data.

Reply:
Should we parameterize this to take in an experimental temperature?

Reply:
probably but lets default it to the IUPAC convention of `298.15 Kelvin` 

https://sciencenotes.org/what-is-room-temperature/#:~:text=Room%20Temperature%20in%20Science%20and%20Industry&text=The%20International%20Union%20of%20Pure,%C2%B0F%2C%20298.15%20K).

Reply:
Done in [ee688cd](https://github.com/proteneer/timemachine/pull/825/commits/ee688cdb3bf110e59c9707fa9e6729df283a9d05) and [c23e492](https://github.com/proteneer/timemachine/pull/825/commits/c23e4926661325266498abfb64c2731bb9c384c8)

Reply:
We don't have to change this now, but should project-wide `DEFAULT_TEMP` also be changed from 300 to 298.15?

Reply:
(Irrelevant to current PR -- moved to Issue https://github.com/proteneer/timemachine/issues/827 for later reminder)

Reply:
yes